sources/niUI/src/Font.cpp

```diff
@@ -1011,7 +1011,7 @@ float cFont::GetCharWidth(tU32 c) const {
   if (c == 0) {
     return 0;
   }
-  return GetCharRectWidth(GetCharTexCoo(' '));
+  return GetCharRectWidth(GetCharTexCoo(c));
```

Comment:
I don't remember if this was intentional. It does look like a bug, but text rendering is wierd and has a lot of edge cases I might have done that on purpose. @ArantirZhang do you have an example of problem that this fix solves? Logically this doesnt look like what you would ever want - there's already GetAverageCharWidth() to get a "somewhat average size" for the font.

Side note: please don't use this to compute text size for anything, use ComputeTextSize(). This function is probably not something you want to ever use in practise, it should probably be deprecated.